tribe-ext-formatted-event-date-shortcode/index.php

```diff
@@ -31,12 +32,13 @@ class Tribe__Extension__Formatted_Event_Date_Shortcode extends Tribe__Extension
 	// Define the shortcode name
 	public $shortcode_name = 'tribe_formatted_event_date';
 	
+	// Define the timezone to display in the output, if any
+	private $timezone_for_display = '';
+	
 	/**
 	 * Setup the Extension's properties.
 	 *
 	 * This always executes even if the required plugins are not present.
-	 * 
-	 * @access: public
 	 */
 	public function construct() {
 		// Each plugin required by this extension
```

Comment:
Do you mind adding set_version here? https://github.com/moderntribe/products-extensions#standards